notebooks/lerobot_act/lerobot-act.ipynb

```diff
@@ -0,0 +1,854 @@
+{
```

Comment:
<p>Could you simplify the PyPI installation? or just recuse a [wrapped function](<a href="https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/utils/pip_helper.py" target="_blank">https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/utils/pip_helper.py</a>) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/3111/discussion/#comment-2600773220'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>